src/components/folder/LinkCards.js

```diff
@@ -1,17 +1,25 @@
-import React from 'react';
+import React, { useState } from 'react';
+
 import addIcon from '../../images/kebab.svg';
 import starIcon from '../../images/star.svg';
 import noImgIcon from '../../images/noCardImg.png';
 import '../../style/folderContent.css';
+import KebabMenu from './KebabMenu';
+
+const LinkCard = ({ link, setModalState }) => {
+  const [kebabVisible, setKebabVisible] = useState(false);
```

Comment:
```suggestion
  const [isVisibleKebab, setIsVisibleKebab] = useState(false);
```

[A/HC/LC Pattern](https://jopemachine.github.io/2021/10/10/Function-Naming-Pattern-A-Hc-Lc-Pattern/)으로 네이밍 컨벤션을 가져가보는건 어떨까요?

실무에서도 많이 사용하는 컨벤션이라 협업시 공통된 네이밍 컨벤션이 있다면 협업할때 코드 가독성이 올라갑니다!